core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -58,6 +58,8 @@ oppia.directive('editorNavigation', [
             $scope.userIsLoggedIn = userInfo.isLoggedIn();
           });
 
+          $scope.can_edit = GLOBALS.can_edit;
```

Comment:
camelCase the js variable names. Also, try not to use GLOBALS. (I think you can use EditableService here.)

Reply:
EditabilityService.isEditable() gives me a false even when I am logged in through the account which has the edit rights.
Also, I've noticed there are 3 types of edit variables. GLOBALS.canEdit, GLOBALS.can_edit, and one in editability service itself. I don't get why we've to define 2 variables for the same task or are there any differences ?

Reply:
@yashdusing, Thanks for looking into the EditabilityService,  the isEditable method returns true if the editor tab is ready for edit i.e, it will be false if someone is going through the tutorial so, It's not just GLOBALS.can_edit.

@vojtechjelinek, have we removed GLOBALS.can_edit or do we have any plans to remove it? if so can you please suggest a way to do it here?

Reply:
Also, many of the other directives use GLOBALS for checking rights as well, so I haven't been able to find alternatives till now. Still searching though

Reply:
Alright. Also, I think GLOBALS.can_edit and GLOBALS.canEdit are actually different variables since can_edit gives me a true for edit rights but canEdit gives me an undefined value :/
Hence, I am using can_edit

Reply:
@yashdusing, I asked you to change $scope.can_edit to $scope.canEdit. (as we use lower camelCasing in js)

Reply:
Oh, my bad. I thought you meant both so I tried both :p

Reply:
Also, I've made the necessary changes, could you please verify the PR ?